cmd/earthly/org_cmds.go

```diff
@@ -448,3 +455,53 @@ func (app *earthlyApp) actionOrgMemberRemove(cliCtx *cli.Context) error {
 
 	return nil
 }
+
+func (app *earthlyApp) actionOrgSelect(cliCtx *cli.Context) error {
+	app.commandName = "orgSelect"
+	if cliCtx.NArg() != 1 {
+		return errors.New("invalid number of arguments provided")
+	}
+	org := cliCtx.Args().Get(0)
+	cloudClient, err := app.newCloudClient()
+	if err != nil {
+		return err
+	}
+
+	allOrgs, err := cloudClient.ListOrgs(cliCtx.Context)
```

Comment:
can `GetOrgID` be used instead? It performs the same `if o.Name == org` logic inside the function.

Reply:
done!